scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -996,10 +978,9 @@ String getTaskServerLogsByTag(String id, String taskTag) throws UnknownJobExcept
      * @throws NotConnectedException
      * @throws PermissionException
      */
-    Page<TaskState> getTaskStates(String taskTag, long from, long to,
-                                  boolean mytasks, boolean running, boolean pending, boolean finished,
-                                  int offset, int limit, SortSpecifierContainer sortParams)
-            throws NotConnectedException, PermissionException;
+    Page<TaskState> getTaskStates(String taskTag, long from, long to, boolean mytasks, boolean running,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.ow2.proactive.scheduler.common.exception.NotConnectedException, org.ow2.proactive.scheduler.common.exception.PermissionException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)